pkg/fioapp/layers.go

```diff
@@ -83,16 +79,49 @@ func GetAppLayersFromMap(ctx context.Context, svcImages map[string]string) (map[
 			return nil, err
 		}
 
-		manifestList, err := GetManifestList(ctx, imageManifestSvc, imageRef.(reference.Canonical).Digest())
+		indexManifest, err := imageManifestSvc.Get(ctx, imageRef.(reference.Canonical).Digest())
 		if err != nil {
 			return nil, err
 		}
 
-		for _, manifest := range manifestList.Manifests {
-			if _, ok := archToManifestList[manifest.Platform.Architecture]; !ok {
-				archToManifestList[manifest.Platform.Architecture] = make(map[distribution.ManifestService]digest.Digest)
+		switch im := indexManifest.(type) {
+		case *manifestlist.DeserializedManifestList:
+			{
```

Comment:
you don't need the brackets here for golang.

Reply:
I know, I just think that using brackets in the case of many lines makes code a bit more readable, and `go lint` doesn't complain, so it must be ok from golang perspective just matter of style :).
Anyway, I was going to move the case code into separate functions, so in this case brackets won't be useful.